apps/movies/src/movies.service.ts

```diff
@@ -16,8 +17,8 @@ export class MoviesService {
     return this.moviesRepository.create(movie);
   }
 
-  findAll() {
-    return this.moviesRepository.find({});
+  async findAll(options: IPaginationOptions): Promise<Pagination<Movie>> {
```

Comment:
```suggestion
  async findAllPaginated(options: IPaginationOptions, searchOptions): Promise<Pagination<Movie>> {
```
As you remember we have 2 tabs for current movies and up comming, so searchOptions will help you to filter movies by conditions

Reply:
To avoid misunderstanding in future when we really will need to get all :D